recipe/windows-bat.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/graphviz/backend.py b/graphviz/backend.py
+index 955551a..be35745 100644
+--- a/graphviz/backend.py
++++ b/graphviz/backend.py
+@@ -69,6 +69,8 @@ def command(engine, format, filepath=None):
+     if format not in FORMATS:
+         raise ValueError('unknown format: %r' % format)
+ 
++    if PLATFORM == 'windows':
++        engine = '%s.bat' % engine
```

Comment:
The advantage of my approach is that `cmd.exe` blank windows won't be shown when making calls to `python-graphviz` from GUIs like Spyder or the Jupyter notebook. That's just a side effect of `shell=True`, but a neat one in this case.

> IMHO, a better solution would be to not have the .bat wrappers

Unfortunately that's not possible at the moment. I created the `graphviz` recipe and decided to use batch scripts to avoid clashes between the libraries provided by `graphviz` and those in other packages. The thing is we're not compiling `graphviz` on Windows (because it's really hard) and instead we just repackage what comes in its MSI installer as a conda package.